components/actionbutton/package.json

```diff
@@ -14,19 +14,17 @@
     "url": "https://github.com/adobe/spectrum-css/issues"
   },
   "main": "dist/index-vars.css",
-  "scripts": {
-    "build": "gulp",
-    "clean": "rimraf dist"
-  },
   "peerDependencies": {
     "@spectrum-css/icon": ">=4",
     "@spectrum-css/tokens": ">=13"
   },
+  "peerDependenciesMeta": {
+    "@spectrum-css/icon": {
+      "optional": true
+    }
+  },
```

Comment:
Since none of our packages include `dependencies`, none of these changes will impact them downstream. All our packages list `devDependencies` (those required for the build process) and `peerDependencies` (expected to be installed by the downstream consumer). We're clarifying the existing `peerDependencies` in this PR to highlight elements that are optional to rendering the component so that downstream users not using those variations don't get unnecessary warnings for missing peer dependencies.